src/registry/aminoTypes.ts

```diff
@@ -14,8 +24,22 @@ export function isAminoMsgDeposit(msg: AminoMsg): msg is AminoMsgDepositDfract {
     return msg.type === 'lum-network/MsgDeposit';
 }
 
-export const createAminoTypes = (): Record<string, AminoConverter> => {
+export const createDefaultAminoTypes = (): { [p: string]: AminoConverter | 'not_supported_by_chain' } => {
     return {
+        ...createBankAminoConverters(),
+        ...createAuthzAminoConverters(),
+        ...createDistributionAminoConverters(),
+        ...createGovAminoConverters(),
+        ...createStakingAminoConverters('lum'),
+        ...createIbcAminoConverters(),
+        ...createFeegrantAminoConverters(),
+        ...createVestingAminoConverters(),
```

Comment:
Confident that all converters are here and not a single ones missing?

Reply:
Yes, at least all we use for Wallet and DFract